scripts/install

```diff
@@ -29,233 +26,162 @@ path_url=$YNH_APP_ARG_PATH
-### guaranteed to be unique. This is a good unique identifier to define installation path,
-### db names, ...
 app=$YNH_APP_INSTANCE_NAME
 
 #=================================================
 # CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
 #=================================================
-ynh_print_info "Validating installation parameters..."
-
-### If the app uses nginx as web server (written in HTML/PHP in most cases), the final path should be "/var/www/$app".
-### If the app provides an internal web server (or uses another application server such as uwsgi), the final path should be "/opt/yunohost/$app"
-final_path=/var/www/$app
-test ! -e "$final_path" || ynh_die "This path already contains a folder"
-
-# Normalize the url path syntax
-path_url=$(ynh_normalize_url_path $path_url)
+ynh_print_info --message="Validating installation parameters..."
 
 # Register (book) web path
-ynh_webpath_register $app $domain $path_url
+ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
-ynh_print_info "Storing installation settings..."
+ynh_print_info --message="Storing installation settings..."
 
-ynh_app_setting_set $app domain $domain
-ynh_app_setting_set $app path $path_url
-ynh_app_setting_set $app is_public $is_public
-ynh_app_setting_set $app nextclouddomain $nextclouddomain
+ynh_app_setting_set --app=$app --key=domain --value=$domain
+ynh_app_setting_set --app=$app --key=path --value=$path_url
+ynh_app_setting_set --app=$app --key=is_public --value=$is_public
+ynh_app_setting_set --app=$app --key=nextclouddomain --value=$nextclouddomain
 
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
 # FIND AND OPEN A PORT
 #=================================================
-ynh_print_info "Configuring firewall..."
-
-### Use these lines if you have to open a port for the application
-### `ynh_find_port` will find the first available port starting from the given port.
-### If you're not using these lines:
-###		- Remove the section "CLOSE A PORT" in the remove script
+ynh_print_info --message="Configuring firewall..."
 
 # Find a free port
-port=$(ynh_find_port 8095)
-# Open this port
-#ynh_exec_warn_less yunohost firewall allow --no-upnp TCP $port
-ynh_app_setting_set $app port $port
+port=$(ynh_find_port --port=8095)
+ynh_app_setting_set --app=$app --key=port --value=$port
 
 #=================================================
 # ADD NODEJS REPOSITORY
 #=================================================
-ynh_print_info "Add nodejs repository..."
+ynh_print_info --message="Add nodejs repository..."
 
-curl -sL https://deb.nodesource.com/setup_8.x | sudo bash -
+ynh_install_extra_repo --repo="deb https://deb.nodesource.com/node_8.x stretch main" --key="https://deb.nodesource.com/gpgkey/nodesource.gpg.key"
```

Comment:
Why do you need to add this repo ?
Can't see any further command to install node.

If you need to install nodeJS, please prefer the helper node instead of installing it manually.

Reply:
nodejs is requested to install onlyoffice but the installation fails when adding nodejs using `ynh_install_nodejs --nodejs_version="8.12" ` or `ynh_install_nodejs --nodejs_version="8" `

Reply:
Probably because when onlyoffice is installed, the shell doesn't use the correct version of node.
Anyway, installing node that way could be source of issues. Be really careful with that.

Reply:
I know that I tryed to be as careful as possible, taking a lot of time trying different way to install onlyoffice with ynh_add_extra_apt_repos__3. But that was the only working option I found. 
Maybe I will be more lucky working with [collabora](https://github.com/YunoHost-Apps/collabora_ynh)